src/prefect/cli/deploy.py

```diff
@@ -691,14 +702,14 @@ async def _run_single_deploy(
                     f" [yellow]{deploy_config_before_templating.get('name')}[/yellow]"
                     " and entrypoint:"
                     f" [yellow]{deploy_config_before_templating.get('entrypoint')}[/yellow]"
-                    " in the [yellow]prefect.yaml[/yellow] file. Would you like to"
-                    " overwrite that entry?"
+                    f" in the [yellow]prefect.yaml[/yellow] file at {prefect_file}."
+                    " Would you like to overwrite that entry?"
                 ),
             ):
                 app.console.print(
                     "[red]Cancelled saving deployment configuration"
                     f" '{deploy_config_before_templating.get('name')}' to the"
-                    " prefect.yaml file.[/red]"
+                    f" prefect.yaml file.[/red] at {prefect_file}"
```

Comment:
```suggestion
                    f" deployment configuration file[/red] at {prefect_file}"
```
I think this is what you mean?